From e9de73bb452e6d8bc46efe5de840abccf76f3c32 Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Thu, 7 Jul 2016 09:43:48 +0200
Subject: [PATCH] fs-lib: add crc32c kernel module for f2fs

(cherry picked from commit f8ff380bd5aadc00061c6537c54ca14728ded6cd)
---
 modules.d/99fs-lib/module-setup.sh | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/modules.d/99fs-lib/module-setup.sh b/modules.d/99fs-lib/module-setup.sh
index a29a350..2c5f4c0 100755
--- a/modules.d/99fs-lib/module-setup.sh
+++ b/modules.d/99fs-lib/module-setup.sh
@@ -20,9 +20,9 @@ echo_fs_helper() {
         ext?)
             echo -n " e2fsck "
             ;;
-	f2fs)
-	    echo -n " fsck.f2fs "
-	    ;;
+        f2fs)
+            echo -n " fsck.f2fs "
+            ;;
         jfs)
             echo -n " jfs_fsck "
             ;;
@@ -41,7 +41,7 @@ echo_fs_helper() {
 include_fs_helper_modules() {
     local dev=$1 fs=$2
     case "$fs" in
-        xfs|btrfs)
+        xfs|btrfs|f2fs)
             instmods crc32c
             ;;
     esac
